Sources/ContainerClient/Utility.swift

```diff
@@ -148,6 +148,20 @@ public struct Utility {
         var mounts = try Parser.mounts(management.mounts)
         mounts.append(contentsOf: tmpfs)
         mounts.append(contentsOf: volumes)
+
+        // Semantic validation: resolve named volume markers to actual paths
```

Comment:
Same comment as above. If we resolve the volumes the second we know it is one, we don't need some sentinel value to tell us later